UPGRADE.md

```diff
@@ -128,6 +142,22 @@ There is a list of all the repositories maintained by monorepo, changes in log b
     - modify your `routing_front.yml`, change configuration for routes `front_image`, `front_image_without_type` - replace `/content/` by `/%shopsys.content_dir_name%/`
 - *(optional)* [#535 added .dockerignore files](https://github.com/shopsys/shopsys/pull/535)
     - if you're using Docker-sync, add the directories mentioned in the PR into `sync_exclude` section of your `docker-sync.yml` to make the synchronization faster
+- [#580 Removed trailing whitespaces from markdown files ](https://github.com/shopsys/shopsys/pull/580)
+    - remove these lines from `.dockerignore`
```

Comment:
`.dockerignore` is in version control, there is no need to modify it when upgrading the monorepo